tao_compiler/mlir/disc/transforms/passes.h

```diff
@@ -234,6 +234,14 @@ createFunctionDeadArgumentEliminationPass();
 std::unique_ptr<OperationPass<gpu::GPUFuncOp>>
 createSideEffectLoopInvariantCodeMotionPass();
 
+// Apply dense gemm/conv to sparse gemm/conv optimizations.
+std::unique_ptr<OperationPass<FuncOp>> createDiscDenseToSparsePass(
```

Comment:
Do we actually need this flag? maybe just not put this pass in the pass pipeline is better if `enable_sparse` is false.

Reply:
Done